backend/CircleForms/Services/PostsService.cs

```diff
@@ -13,19 +14,24 @@
 using CircleForms.Models.Posts.Questions;
 using CircleForms.Models.Posts.Questions.Submissions;
 using CircleForms.Services.Database.Interfaces;
+using CircleForms.Services.IO;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 
 namespace CircleForms.Services;
 
 public class PostsService
 {
+    private readonly IStaticFilesService _cdnResolveService;
```

Comment:
cringe naming but Okay

Reply:
it shouldn't be here, it was a previous name :/